### PR TITLE
Do not override local modifications by default

### PR DESCRIPTION
**Pull request by wrabcak**
Following PR resolves: 
https://bugzilla.redhat.com/show_bug.cgi?id=1597545

### CODE REVIEW

**Comment by bachradsusi** [comment]
I have only cosmetic issues with PR:

I would squash d407864 and d9921ca together.

`selinux_all_purge` is not documented in commit message and none of new variables is documented in README.

It could be useful to have also tests to cover new variables and new functionality and to be sure that we don't override system modifications.

Otherwise it looks good.

**Comment by wrabcak** [comment]
> I would squash d407864 and d9921ca together.
Fixed.

> selinux_all_purge is not documented in commit message and none of new variables is documented in README.
Fixed.

> It could be useful to have also tests to cover new variables and new functionality and to be sure that we don't override system modifications.

Let's create issue for missing tests.

File: README.md [hunk 1]
```diff
@@ -45,6 +45,15 @@ roles:
...
 
+#### purge local modifications using appropriate variable
+
+```yaml
+selinux_all_purge: 1
+selinux_booleans_purge: 1
+selinux_fcontexts_purge: 1
+selinux_ports_purge: 1
+selinux_logins_purge: 1
+```
```
Comment (bachradsusi) [inline README.md]: There were some comments about using boolean True/False instead of numbers. Should this be the case?

Comment (wrabcak) [reply README.md]: Fixed.

**Comment by pcahyna** [comment]
[citest bad]

File: README.md (hunk 1 above)
Comment (pcahyna) [reply README.md]: having `selinux_all_purge: true` with the other `*purge` variables in the same yaml example is IMO confusing. Can you split it into two example files, one with `selinux_all_purge`, the other with the others?

**Comment by wrabcak** [comment]
Looks like this PR is ready. Will rebase and merge.

**Pull request merged**